deploy/infrastructure/modules/terraform-aws-dss/README.md

```diff
@@ -0,0 +1,74 @@
+# terraform-aws-dss
+
+This terraform module creates a Kubernetes cluster in Amazon Web Services using the Elastic Kubernetes Service (EKS) 
+and generates the tanka files to deploy a DSS instance.
+
+
+## Getting started
+
+### Prerequisites
+
+Download & install the following tools to your workstation:
+
+1. Install [terraform](https://developer.hashicorp.com/terraform/downloads).
+2. Install tools from [Prerequisites](../../../../build/README.md)
+3. Install provider specific tools:
+    1. [Amazon Web Services](./README.md#amazon-web-services)
+
+    
+#### Amazon Web Services
+
+1. Install and initialize [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions).
+    1. Confirm successful installation with `aws --version`.
+2. If you don't have an account, sign-up: https://aws.amazon.com/free/
+3. Configure terraform to connect to AWS using your account.
+   1. We recommend to create an AWS_PROFILE using for instance `aws configure --profile aws-interuss-dss`
+      Before running `terraform` commands, run once in your shell: `export AWS_PROFILE=aws-interuss-dss`
+      Other methods are described here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration
+   
+
+### Deployment of the Kubernetes cluster
+
+1. Create a new folder in `/deploy/infrastructure/personal/` named, for instance, `terraform-aws-dss-dev`.
+2. Copy main.tf, output.tf and variables.tf to the new folder.
+3. Copy `terraform.dev.example.tfvars` and rename to `terraform.tfvars`
+4. Check that your new directory contains the following files:
+   - main.tf
+   - output.tf
+   - terraform.tfvars
+   - variables.tf
+5. Set the variables in `terraform.tfvars` according to your environment. See [TFVARS.md](TFVARS.md) for variables descriptions.
+6. In the new directory (ie /deploy/infrastructure/personal/terraform-aws-dss-dev), initialize terraform: `terraform init`.
+7. Run `terraform plan` to check that the configuration is valid. It will display the resources which will be provisioned.
+8. Run `terraform apply` to deploy the cluster. (This operation may take up to 15 min.)
+9. If `aws_route53_zone_id` was not provided, configure the DNS resolution to the public ip addresses and for SSL certificate validation.  
```

Comment:
I would recommend slightly rewording this: if `aws_route53_zone_id` is provided, please make sure that the domain or sub-domain in the Route 53 zone is mapped to Route53 via NS entries, e.g. see instuctitions for a [sub-domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html#UpdateDNSParentDomain)